booth.spec

```diff
@@ -1,5 +1,11 @@
 %bcond_with html_man
 
+%if 0%{?fedora} > 18 || 0%{?centos} > 6 || 0%{?rhel} > 6
```

Comment:
Oops, it was and issue in `configure.ac` and not setting defaults if `--without-X` not provided.